shared/chat/conversation/messages/attachment/image/index.js

```diff
@@ -55,12 +56,16 @@ class ImageAttachment extends React.PureComponent<Props, State> {
         <Box
           style={collapseStyles([
             styles.loading,
+            !this.state.loaded ? styles.spinner : {},
             {
               height: this.props.height,
               width: this.props.width,
             },
           ])}
         >
+          {!this.state.loaded && (
+             <ProgressIndicator style={{margin: 'auto'}} />
+          )}
           {!!this.props.path && (
```

Comment:
If looks like if we do `!this.state.loaded ? ....`, this block can just be part of what's after `:`. Then we don't need `display: none` and can get rid of `opacity` below.